intake/catalog/local.py

```diff
@@ -120,21 +118,19 @@ def __init__(self, name, description, driver, direct_access, args,
         self._user_parameters = parameters
         self._metadata = metadata
         self._catalog_dir = catalog_dir
-        self._plugin = None
+        self._plugin = global_registry[driver]
         super(LocalCatalogEntry, self).__init__(
             getenv=getenv, getshell=getshell)
 
     @property
     def name(self):
         return self._name
 
-    def find_plugin(self, registry):
-        if self._driver in registry:
-            self._plugin = registry[self._driver]
-        else:
-            self._plugin = global_registry[self._driver]
-
     def describe(self):
+        if isinstance(self._plugin.container, (list, tuple)):
+            import pdb
+            pdb.set_trace()
+            print('####', self._plugin)
```

Comment:
Perhaps we should have an issue to come back and replace `print` with a logger?

Reply:
done